src/app/jumio/jumio-error/jumio-error.component.html

```diff
@@ -0,0 +1,17 @@
+<app-banner></app-banner>
+<div class="container home-container">
+  <div class="fs-24px font-weight-bold pt-15px">Jumio Verification Failed</div>
+  <div class="d-flex align-items-center">
+    <div class="fs-15px mt-15px">
+      We were unable to verify your identity. You can click close to proceed to {{ hostname }}
+    </div>
+  </div>
+  <div class="d-flex pt-15px justify-content-between">
+    <button
+      (click)="finishFlow()"
+      class="btn btn-outline-primary font-weight-bold fs-15px sign-up-btn"
+    >
+     Close
```

Comment:
This is maybe a bit confusing. Perhaps it should be "Continue to {{ hostname }}" just like the end of the Google flow?

Reply:
I like that. I'll make that change